cmd/incusd/api_project.go

```diff
@@ -1541,3 +1566,93 @@ func projectValidateRestrictedSubnets(s *state.State, value string) error {
 
 	return nil
 }
+
+// Do I need to check the StatusCode and/or ExpandedConfig similar to "cmd/incus/delete.go"
```

Comment:
No, that shouldn't apply to an internal call., but you do need to make sure the instance is stopped before you delete it.

Reply:
fixed code to check if its running and stop